webapp/channels/src/components/edit_post/edit_post.tsx

```diff
@@ -1,671 +1,45 @@
-                >
-                    <EmoticonPlusOutlineIcon
-                        size={18}
-                        color='currentColor'
-                    />
-                </button>
-            </>
-        );
+    if (!editingPostDetailsAndPost.show) {
+        return null;
     }
 
-    let rootId = '';
-    if (editingPost.post) {
-        rootId = editingPost.post.root_id || editingPost.post.id;
-    }
+    const channelId = editingPostDetailsAndPost.post.channel_id;
+    const location = editingPostDetailsAndPost.isRHS ? Locations.RHS_COMMENT : Locations.CENTER;
+    const rootId = editingPostDetailsAndPost.post.root_id || '';
+    const storageKey = `${StoragePrefixes.EDIT_DRAFT}${editingPostDetailsAndPost.post.id}`;
 
     return (
-        <div
-            className={classNames('post--editing__wrapper', {
-                scroll: renderScrollbar,
-            })}
-            ref={wrapperRef}
-        >
-            <Textbox
-                tabIndex={0}
-                rootId={rootId}
-                onChange={handleChange}
-                onKeyPress={handleEditKeyPress}
-                onKeyDown={handleKeyDown}
-                onBlur={handleBlur}
-                onHeightChange={handleHeightChange}
-                handlePostError={handlePostError}
-                onPaste={handlePaste}
-                value={editText}
+        <div className='post-edit__container'>
+            <AdvancedTextEditor
+                location={location}
                 channelId={channelId}
-                emojiEnabled={config.EnableEmojiPicker === 'true'}
-                createMessage={formatMessage({id: 'edit_post.editPost', defaultMessage: 'Edit the post...'})}
-                supportsCommands={false}
-                suggestionListPosition='bottom'
-                id='edit_textbox'
-                ref={textboxRef}
-                characterLimit={rest.maxPostSize}
-                useChannelMentions={rest.useChannelMentions}
-            />
-            <div className='post-body__actions'>
-                {emojiPicker}
-            </div>
-            { showMentionHelper ? (
-                <div className='post-body__info'>
-                    <span className='post-body__info__icon'>
-                        <InformationOutlineIcon
-                            size={14}
-                            color='currentColor'
-                        />
-                    </span>
-                    <span>{
-                        formatMessage({
-                            id: 'edit_post.no_notification_trigger_on_mention',
-                            defaultMessage: "Editing this message with an '@mention' will not notify the recipient.",
-                        })
-                    }</span>
-                </div>) : null
-            }
-            <EditPostFooter
-                onSave={handleEdit}
-                onCancel={handleCancel}
+                postId={rootId}
+                isThreadView={false}
```

Comment:
Should this be set to true if this post is in the Threads view?

Reply:
this prop is passed explicitly when you know this will be opened in the RHS, but the editing post will figure that out so removing from here